ui/src/containers/NodeVolumes.js

```diff
@@ -40,8 +39,15 @@ import {
 const VolumeTable = styled.div`
   flex-grow: 1;
   margin-top: ${padding.small};
-  .sc-table-column-cell-container-action {
+  /* .sc-table-column-cell-container-action {
```

Comment:
Zombie code, kill it! :stuck_out_tongue: 

We should avoid keeping commented out code: if it's useless remove it (it's not lost, we have the Git history).

See also https://www.bitnative.com/2012/10/22/kill-the-zombies-in-your-code/